### PR TITLE
Expanding and cleaning up weight check metrics.

### PR DESCRIPTION
**Pull request by michael-mclawhorn**
Four corner metrics establish lines scanned, returned, sketches scanned, returned.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java [hunk 1]
```diff
@@ -74,19 +74,23 @@
  *    "aggregations": [
  *      {
  *        "name": "resultSketches",
- *        "fieldName": "rawSketches",
+ *        "fieldName": "sketchesPerRow",
  *        "type": "longSum"
  *      },
```
Comment (will-lauer) [inline fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java:79]: Why is this here if its just overriden in the postAgg section?

**Review by will-lauer — APPROVED** [review]

File: fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java (hunk 1 above)
Comment (michael-mclawhorn) [reply fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java:79]: The fieldname is the downward pointing reference, so resultSketches is the total of the sketchesPerRow from the inner layer.

Comment (will-lauer) [reply fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java:79]: ah, I missed that I was looking at fieldname rather than just `name`. makes sense now.
